uhabits-core/build.gradle.kts

```diff
@@ -30,6 +30,8 @@ kotlin {
             dependencies {
                 implementation(kotlin("stdlib-common"))
                 implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core-common:1.3.8")
+                implementation("org.jetbrains.kotlinx:kotlinx-datetime:0.2.1")
+                implementation("io.fluidsonic.locale:fluid-locale:0.10.0")
```

Comment:
@iSoron OK to use this library for Locale? Or do we need to role our own per platform?

Reply:
It looks like the library's main functionality is parsing language tags (e.g. "ZH-HANT-cn-somevar"). Unless I am missing something, I don't think we need this functionality; we can just store the tag as is. For example, it seems to me like the following `Locale` class would suffice for our purposes, but please let me know if I'm mistaken:

```kotlin
data class Locale(val tag: String)
```